metagraph/src/annotation/annotation_converters.cpp

```diff
@@ -1184,6 +1185,15 @@ void convert_to_row_diff(const std::vector<std::string> &files,
         for ( ; i < files.size(); ++i) {
             // also add some space for buffers for each column
             uint64_t file_size = fs::file_size(files[i]) + ROW_DIFF_BUFFER_BYTES;
+            if (with_values) {
+                // also add k-mer counts
+                try {
+                    const auto &values_fname
```

Comment:
why not use `replace_extension`?

Reply:
std::filesystem::replace_extension removed only a single `.XXX` extension, but here we have two: `.column.annodbg`.